framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenes.java

```diff
@@ -606,32 +606,32 @@ protected boolean ignoreFieldInWPI(Element element, String fieldName) {
     }
 
     /**
-     * Updates the return type of the method methodTree in the Scene of the class with symbol
+     * Updates the return type of the method methodDecl in the Scene of the class with symbol
      * classSymbol. Also updates the return types of methods that this method overrides, if they are
      * available as source.
      *
-     * <p>If the Scene does not contain an annotated return type for the method methodTree, then the
+     * <p>If the Scene does not contain an annotated return type for the method methodDecl, then the
      * type of the value passed to the return expression will be added to the return type of that
      * method in the Scene. If the Scene previously contained an annotated return type for the
-     * method methodTree, its new type will be the LUB between the previous type and the type of the
+     * method methodDecl, its new type will be the LUB between the previous type and the type of the
      * value passed to the return expression.
      *
      * @param retNode the node that contains the expression returned
      * @param classSymbol the symbol of the class that contains the method
-     * @param methodTree the tree of the method whose return type may be updated
+     * @param methodDecl the declaration of the method whose return type may be updated
      * @param overriddenMethods the methods that the given method return overrides, each indexed by
      *     the annotated type of the class that defines it
      */
     @Override
     public void updateFromReturn(
             ReturnNode retNode,
             ClassSymbol classSymbol,
-            MethodTree methodTree,
+            MethodTree methodDecl,
```

Comment:
I prefer `methodTree`.  A method declaration can also be represented by an element, so it's not clear whether a variable name `methodDecl` is a tree or an element..  But you can leave it if you feel strongly.

Reply:
@smillst I found it confusing to know from the name `methodTree` whether it is a method *call* or method *declaration*.  How about `methodDeclTree`?

Reply:
That works for me.